components/statistics/BasicCards.js

```diff
@@ -9,7 +9,7 @@ export default function BasicCards({ data }) {
         (item) =>
           item.current > 0 && (
             <div
-              className="overflow-hidden rounded-lg bg-white dark:bg-primary-medium px-4 py-5 shadow sm:p-6"
+              className="hidden md:block overflow-hidden rounded-lg bg-white dark:bg-primary-medium px-4 py-5 shadow sm:p-6"
```

Comment:
this will hide the card on mobile wherever it is used, we only want it hidden on the profile page

Reply:
```suggestion
              className="overflow-hidden rounded-lg bg-white dark:bg-primary-medium px-4 py-5 shadow sm:p-6"
```